pipelines/coaddDiffMatchedInjectedCore.yaml

```diff
@@ -0,0 +1,201 @@
+description: |
+  Matched difference (measured vs reference) plots/metrics
+parameters:
```

Comment:
I think more of this should be in the atool and less in the pipeline file.

Reply:
These are here to make it easier to change settings for all of the relevant plots at once, which is something I thought users would be reasonably likely to want to do.

On the other hand, I don't really like pipeline parameters either. Doing it this way means they have to be the same for all of the plots; if anyone wants to change a few here or there, they'd have to copy the file and change them per-tool anyway. There's no easy way I can think of to keep these in sync with the actual config defaults either.

Reply:
See the note below. I dropped the parameters entirely.

I could re-implement the ability to mass change limits etc. in `reconfigure_diff_matched_defaults` if needed.

This file also ends up doing the same thing as the original DC2 `/coaddDiffMatchedQualityExtended.yaml` pipeline, which I have not updated here. They could be consolidated at some point, or have one base that they each import.